src/main/configurations/Translate/Configuration_Zaken_GetZgwRolTypeByUrl.xml

```diff
@@ -20,12 +20,12 @@
 					name="GetZgwRolTypeByUrlHttpSender" 
 					methodType="GET"
 					headersParams="Authorization,Accept-Crs"
-                    timeout="${creeerZaak.timeout}"
+                    timeout="${zaakbrug.zgw.catalogi-api.timeout}"
```

Comment:
Why is the indentation so wierd? Is it a GitHub thing or is it like this in de configuration as well?